Google.PowerShell.IntegrationTests/CloudResourceManager/CloudResourceManager.IamPolicy.Tests.ps1

```diff
@@ -23,16 +23,25 @@ Describe "Get-GcIamPolicyBinding" {
 }
 
 Describe "Add-GcIamPolicyBinding" {
+    # Returns true if binding for member $member to role $role exists in project $projectToCheck.
+    function Test-Binding($member, $role, $projectToCheck) {
+        $bindings = Get-GcIamPolicyBinding -Project $projectToCheck
+        $binding = $bindings | Where-Object {$_.Role -eq $role}
+        if ($null -eq $binding -or $null -eq $binding.Members) {
+            return $false
+        }
+        return ($binding.Members -contains $member)
+    }
+
     It "should work for -User" {
         $role = "roles/browser"
         $user = "quoct@google.com"
```

Comment:
I'm still a little uncomfortable with using real users and groups.  Is this required?  if so maybe make some for this test?